HECalculator/Backend/Worker.cs

```diff
@@ -4,6 +4,9 @@ public class Worker
 {
 	public Worker()
 	{
+		this.Inputs=new Inputs();
+		this.Validations=new Validations(States);
```

Comment:
Swap with line 8 and 9.
In this case States is pass as parameter to Validation with value null. It is common to first initialize variable and then pass it. Anyway, it will work OK.